### ISSUE TITLE
Add cache_valid_time to 1.1.1 

### ISSUE THREAD

**Issue by systeminsightsbuild** [awailly/cis-ubuntu-ansible#87]
Maybe:

```
    apt: update_cache=yes cache_valid_time={{apt_cache_valid_time}}
```

With `apt_cache_valid_time: 3600` in defaults/main.yml

As part of our base playbook we already update the cache. Setting a valid time in 1.1.1 would prevent from updating the cache again. I've seen lots of larger playbooks also update the cache, so this should be generally useful.

**Comment by pchaigno** [comment]
:+1: 

A 1 hour cache seems reasonable to me. @awailly ?

**Comment by awailly** [comment]
Good for me!

**Comment by pchaigno** [comment]
Resolved in #88.
